src/app/showcase/calendar-showcase/calendar-showcase.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ShowcaseProperty } from '~/app/shared/showcase-properties/showcase-property';
+
+declare var require: any;
+
+@Component({
+  selector: 'kirby-calendar-showcase',
+  templateUrl: './calendar-showcase.component.html',
+  styleUrls: ['./calendar-showcase.component.scss'],
+})
+export class CalendarShowcaseComponent {
+  selectValue = 'red';
```

Comment:
red ???

Reply:
I have no clue where that come from 🤔  Must be a copy-paste gone wrong 💩 
Fixed!